src/core/Api.js

```diff
@@ -15,6 +15,16 @@ const API = {
             method: 'post',
             withCredentials: true,
         }),
+    attendConference: (eventId, token) =>
+        axios.post(
+            `https://api.conflist.devlabs-projects.com/api/v1/conferences/${eventId}/attend`,
```

Comment:
Seems like a good time to declare `https://api.conflist.devlabs-projects.com/api/v1` string as API.baseUrl propety. Does it sound good to you?